Library/Source/NativeEngine.cpp

```diff
@@ -714,16 +751,7 @@ namespace babylon
         const auto blendMode = static_cast<BlendMode>(info[0].As<Napi::Number>().Int32Value());
 
         m_engineState &= ~BGFX_STATE_BLEND_MASK;
-        switch (blendMode)
-        {
-        case 0:
-            break;
-        case 25:
-            m_engineState |= BGFX_STATE_BLEND_NORMAL;
-            break;
-        default:
-            throw std::exception("Unsupported blend mode");
-        }
+        m_engineState |= ALPHA_MODE[blendMode];
```

Comment:
throw when alphamode returns 0?

Reply:
Resolved by adding implementation of alpha interpolate.